lib/tape/ansible_runner.rb

```diff
@@ -1,48 +1,54 @@
+class Something
+  def method(arg1, arg2)
```

Comment:
Unused method argument - `arg2`. If it's necessary, use `_` or `_arg2` as an argument name to indicate that it won't be used.